### ISSUE TITLE
DynamicBatteryStorage 2:1.3.2.0 conflicts with RP-0 v1.11.2.0

### ISSUE THREAD

**Issue by vandenbergmp** [KSP-RO/RP-0#1586]
I've been following the Installation Guide for RP-0 express installation to the letter and cant even get the Mod to install. 

https://github.com/KSP-RO/RP-0/wiki/Installation

Here is the status log:

The following inconsistencies were found:
* DynamicBatteryStorage 2:1.3.2.0 conflicts with RP-0 v1.11.2.0
Error during installation!
If the above message indicates a download error, please try again. Otherwise, please open an issue for us to investigate.
If you suspect a metadata problem: https://github.com/KSP-CKAN/NetKAN/issues/new/choose
If you suspect a bug in the client: https://github.com/KSP-CKAN/CKAN/issues/new/choose

Here are the steps I have taken so far:
- Fresh KSP 1.10.1 with expansions removed and launched separately from steam
- Fresh CKAN install with all mods/settings/caches purged (up to date)
- Reinstall CKAN multiple times with multiple browsers

I'm not convinced this is my fault, there may be an issue with the latest version of RP-0. 

Forgive me if this issue has already been addressed but I can't find anything on the subject. 

Please save me geniuses of this community!

**Comment by Capkirk123** [comment]
That is because you tried to install Near Future Electrical, or some other mod which requires Dynamic Battery Storage. If you want to use near future electrical, you will have to install it manually.

**Comment by vandenbergmp** [comment]
Incorrect. I am trying to Install RP-0 Express. If it is installing a mod that shouldn't be there it is an Error in the RP-0 Mod pack itself.

**Comment by Capkirk123** [comment]
Is dynamic battery storage installed?

**Comment by vandenbergmp** [comment]
I'm not sure If you are understanding my problem. The RP-1 Express mod is not installing because of an error. The error is DynamicBatteryStorage 2:1.3.2.0 conflicts with RP-0 v1.11.2.0.

I am not trying to install either of those necessarily, I am only trying to install "RP-1 Express" I suspect the Dev made a mistake and has incompatible mods in the mod list, or I have my CKAN messed up somehow. If this were a manual install then yes, you would be right. 

As of trying to install RP-1 Express, I have a clean install of 1.10.1 KSP. No mods prior.

**Comment by Capkirk123** [comment]
RP-1 express install should not be attempting to install Dynamic Battery storage, and does not rquire or recommend it. I guess you should just try again with a clean install.

**Comment by vandenbergmp** [comment]
I am aware of that, but I cannot figure out why. I have downloaded CKAN again, purged the cache (again), and it seems to be working so far. 

There was an extra step in CKAN to install the mod that isn't in the guide and wasn't present before. I am very confused and not exactly thrilled with CKAN right now. I probably could have done a manual realism overhaul myself in the time I've spent messing around with this. 

CKAN seems to be picking random mods that I have downloaded in the past maybe? I've never really used CKAN with other KSP mod sets I've made before, or even connected CKAN to any other instance of KSP on my PC.
